versioned_docs/_version-v1.19.x/user-guide/systemrequirements.md

```diff
@@ -59,9 +59,9 @@ For more information on the relationship between the Zowe server components and
 
 ### Zowe Web Explorers and APIs on z/OS requirements (host)
 
-- Node.js v8.x (except v8.16.1), v12.x, or v14
+- Node.js v14.x(except v14.17.2) or v16.x
```

Comment:
Missing a space between `v14.x` and `(except v14.17.2)`. Check other references as well. : )